sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -74,7 +74,8 @@ public BlobClientOptions GetOptions(bool parallelRange = false)
                     Mode = RetryMode.Exponential,
                     MaxRetries = Storage.Constants.MaxReliabilityRetries,
                     Delay = TimeSpan.FromSeconds(Mode == RecordedTestMode.Playback ? 0.01 : 1),
-                    MaxDelay = TimeSpan.FromSeconds(Mode == RecordedTestMode.Playback ? 0.1 : 60)
+                    MaxDelay = TimeSpan.FromSeconds(Mode == RecordedTestMode.Playback ? 0.1 : 60),
+                    NetworkTimeout = TimeSpan.FromSeconds(200),
```

Comment:
Does `NetworkTimeout` apply to `RecordedTestMode.Playback`?  If so, you might want to use a smaller value for playback.

Reply:
I think that it isn't very relevant there as everything happens in memory. But why not.